pyfolio/perf_attrib.py

```diff
@@ -225,7 +237,8 @@ def plot_factor_contribution_to_perf(exposures, perf_attrib_data, ax=None):
         perf_attrib_data.index,
         [perf_attrib_data[s] for s in chain(perf_attrib_data.iloc[:, :-3],
                                             ['specific_returns'])],
-        labels=list(perf_attrib_data.iloc[:, :-3]) + ['specific returns']
+        labels=list(perf_attrib_data.iloc[:, :-3]) + ['specific returns'],
```

Comment:
why are the last three dropped?

Reply:
the last three columns are `total_returns`, `common_returns`, and `specific_returns`, and common returns are accounted for by each of the individual factor returns

Reply:
maybe `perf_attrib_data.drop(['total_returns', ...], axis='columns')` then to make the intent more clear?